plugins/src/main/java/org/smartregister/fhir/gateway/plugins/Constants.java

```diff
@@ -15,9 +15,15 @@ public class Constants {
     public static final String PARAM_VALUES_SEPARATOR = ",";
     public static final String PROXY_TO_ENV = "PROXY_TO";
     public static final String TAG_SEARCH_PARAM = "_tag";
+    public static final String TYPE_SEARCH_PARAM = "type";
+    public static final String DEFAULT_ADMIN_LEVEL_TYPE_URL =
+            "https://smartregister.org/codes/administrative-level";
```

Comment:
Not sure about this url, considering this comment https://github.com/onaio/fhir-web/issues/1349#issuecomment-2033718095. What url is best to go with `.../codes/..` or `.../CodeSystem/...`

Reply:
@lincmba let's go with `CodeSystem` since this is what web is using

We will update and document that